engine/runner/src/main/scala/org/enso/runner/ContextFactory.scala

```diff
@@ -51,12 +52,22 @@ class ContextFactory {
     executionEnvironment.foreach { name =>
       options.put("enso.ExecutionEnvironment", name)
     }
+    var javaHome = System.getenv("JAVA_HOME");
+    if (javaHome == null) {
+      javaHome = System.getProperty("java.home");
+    }
+    if (javaHome == null) {
+      throw new IllegalStateException("Specify JAVA_HOME environment property");
+    }
     val logLevelName = Converter.toJavaLevel(logLevel).getName
     val builder = Context
       .newBuilder()
       .allowExperimentalOptions(true)
       .allowAllAccess(true)
-      .allowHostAccess(new HostAccessFactory().allWithTypeMapping())
+      .allowHostAccess(
+        new HostAccessFactory()
```

Comment:
spurious change?

Reply:
I haven't touched the line myself. Let's blame `scalafmtAll`! However the CI is green - e.g. the formatting is appropriate. Anyway semantically there is no change and I space-to-space exact formatting is the last thing I care about.